keps/0007-diagnostic-template.md

```diff
@@ -0,0 +1,43 @@
+---
+kep-number: 7
+title: Diagnostic Template
+authors:
+  - "@gerred"
+  - "@meichstedt"
+owners:
+  - "@gerred"
+  - "@meichstedt"
+creation-date: 2019-04-11
+last-updated: 2019-04-11
+status: provisional
+---
+
+# KUDO Diagnostic Template
+
+- [KUDO Diagnostic Template](#kudo-diagnostic-template)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+  - [Proposal](#proposal)
+
+## Summary
+
+The KUDO Diagnostic Template is intended to provide an opinionated template for issues when using frameworks for KUDO and creating frameworks with KUDO. This is intended to supersede `ISSUE_TEMPLATE.md` and provide an opinionated set of diagnostic information, logs, and tooling required to adequately diagnose and respond to issues with KUDO.
```

Comment:
Do we mean the [ISSUE_TEMPLATE](https://github.com/kudobuilder/kudo/tree/4b43a7f54aaa90fb7c71279bc019953ea94e6a29/.github/ISSUE_TEMPLATE) folder or how I read out here more the `bug_report.md` within this folder? The `bug_report.md` started out already opinionated and we can surely modify as we see fit with e.g. empirical issues that arise but I don't really see the need for an extra KEP that defines this template. This came up today in the KUDO Community meeting and was lightly discussed. What think would help if we would have at least one or two prototype-ish outcomes from this KEP so we see it more concreter.